circleci: failed tests: TestDockerFinagle

The following test appears to have failed:

[#21029](https://circleci.com/gh/cockroachdb/cockroach/21029):

```
    at org.junit.runner.JUnitCore.run(JUnitCore.java:117)
    at org.junit.runner.JUnitCore.runMain(JUnitCore.java:98)
    at org.junit.runner.JUnitCore.runMainAndExit(JUnitCore.java:53)
    at org.junit.runner.JUnitCore.main(JUnitCore.java:45)

FAILURES!!!
Tests run: 12,  Failures: 10

I160805 14:27:54.455669 acceptance/cluster/localcluster.go:671  stopping
I160805 14:27:54.455878 acceptance/cluster/localcluster.go:571  event stream done, resetting...: net/http: request canceled
I160805 14:27:54.455896 acceptance/cluster/localcluster.go:594  events monitor exits
--- FAIL: TestDockerFinagle (31.49s)
    util_test.go:386: non-zero exit code: 1
=== RUN   TestFreezeCluster
--- SKIP: TestFreezeCluster (0.00s)
    freeze_test.go:40: #7957
=== RUN   TestGossipPeerings
I160805 14:27:55.846505 acceptance/cluster/localcluster.go:299  Initializing Cluster AdHoc 3x1:
{"name":"AdHoc 3x1","nodes":[{"count":3,"stores":[{"count":1,"max_ranges":0}]}],"duration":5000000000}
I160805 14:27:55.848407 acceptance/cluster/docker.go:106  ImagePull cockroachdb/builder:20160804-184345 already exists
I160805 14:27:56.344613 acceptance/cluster/localcluster.go:605  creating certs (1024bit) in: /home/ubuntu/cockroach/.localcluster.certs.322864832
I160805 14:27:56.433125 acceptance/cluster/localcluster.go:556  events monitor starts
--
--- SKIP: TestBuildBabyCluster (0.00s)
    terraform_test.go:30: only enabled during testing
=== RUN   TestFiveNodesAndWriters
--- SKIP: TestFiveNodesAndWriters (0.00s)
    util_test.go:357: skipping since not run against remote cluster
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```

Please assign, take a look and update the issue accordingly.

--- a/acceptance/finagle_test.go
+++ b/acceptance/finagle_test.go
@@ -16,7 +16,9 @@ package acceptance
 
 import "testing"
 
+// This runs the `finagle-postgres` tests from the upstream project.
 func TestDockerFinagle(t *testing.T) {
+	t.Skip("#8332. Upstream has a 2s timeout, disabled until we run tests somewhere more consistent.")
 	testDockerSuccess(t, "finagle", []string{"/bin/sh", "-c", finagle})
 }
 

